server/game/basecard.js

```diff
@@ -170,15 +171,14 @@ class BaseCard {
         this.abilities.reactions.push(reaction);
     }
 
-    // TODO: When revealed abilities shouldn't be a synonym for forced reactions
-    //       but it is probably close enough for now.
     whenRevealed(properties) {
```

Comment:
I think I understand why, but I find it a little weird that this is on basecard and not plotcard - AFAIK, draw cards don't/can't have 'when revealed's